packages/api-server/api_server/routes/fleets.py

```diff
@@ -180,3 +181,39 @@ async def recommission_robot(
         logger.error(f"Failed to recommission {robot_name} of {name}")
         raise HTTPException(400, resp)
     return resp
+
+
+@router.post("/{name}/unlock_mutex_group")
+async def unlock_mutex_group(
+    name: str,
+    robot_name: str,
+    mutex_group: str,
+    repo: FleetRepository = Depends(fleet_repo_dep),
+    user: User = Depends(user_dep),
+):
+    """
+    Request to manually unlock a mutex group that is currently being held by a
+    specific robot of a specific fleet.
+    """
+    fleet_state = await repo.get_fleet_state(name)
+    if fleet_state is None:
+        raise HTTPException(404, f"Fleet {name} not found")
+    if fleet_state.robots is None or robot_name not in fleet_state.robots:
+        raise HTTPException(404, f"Robot {robot_name} not found in fleet {name}")
+    mutex_groups = fleet_state.robots[robot_name].mutex_groups
+    if (
+        mutex_groups is None
+        or mutex_groups.locked is None
+        or mutex_group not in mutex_groups.locked
+    ):
+        raise HTTPException(
+            404,
```

Comment:
400 error would be more appropriate here.

Reply:
https://github.com/open-rmf/rmf-web/pull/933/commits/8da155b9f1ae8f1a248eba852ba82be9d0061468